### PR TITLE
Sql core and firewall swagger changes

### PR DESCRIPTION
**Pull request by nathannfan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 
### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).
### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [ ] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---

This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/622)

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@nathannfan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nafan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: arm-sql/compositeSql.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "info": {
+    "title": "SqlManagementClient",
+    "description": "Composite Swagger for Azure Sql Database Client"
+  },
+  "documents": [
+    //"./2015-05-01/swagger/jobaccount.json",
```
Comment (anudeepsharma) [inline arm-sql/compositeSql.json]: Minor - This file is having reference to all other files, which are not in this review now

Comment (nathannfan) [inline arm-sql/compositeSql.json]: #Resolved

---
In reply to: [84566104](https://github.com/Azure/azure-rest-api-specs/pull/622#discussion_r84566104) [](ancestors = 84566104)

Comment (amarzavery) [inline arm-sql/compositeSql.json]: @nathannfan - We still see the commented line in the composite document. Can you please make sure to remove this line?
> //"./2015-05-01/swagger/jobaccount.json",

File: arm-sql/2014-04-01/firewallRules.json [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+    },
+    "ServerFirewallRule": {
+      "properties": {
+        "kind": {
+          "type": "string",
+          "readOnly": true
+        },
+        "location": {
+          "type": "string",
+          "readOnly": true
```
Comment (nathannfan) [inline arm-sql/2014-04-01/firewallRules.json]: #Resolved

---
In reply to: [84579657](https://github.com/Azure/azure-rest-api-specs/pull/622#discussion_r84579657) [](ancestors = 84579657)

File: arm-sql/compositeSql.json (hunk 1 above)
Comment (nathannfan) [inline arm-sql/compositeSql.json]: #Resolved

---
In reply to: [84755024](https://github.com/Azure/azure-rest-api-specs/pull/622#discussion_r84755024) [](ancestors = 84755024)

**Pull request merged**